README.md

```diff
@@ -1,2 +1,110 @@
 # Spice
 Small shopping platform that takes customer orders and saves them into a database.
+
+# Product Order
+
+## Product Order Pseudo-code:
+
+First, I’ll create a class that will capture the order-objects attributes. When shopping in a supermarket, customers pick products of certain brands and then decide on the product weight they intend to purchase. That means that an order(object) has attributes like product, weight, and cost. One unique feature about products though is that they have fixed, attributes. 
+
+
+## Order:
+1. Product
+2. Weight
+3. Cost
+
+
+
+I’ll start out by creating an order function then class which define the object's characteristics: 
+
+```python
+def Order():
+	class Order:
+		def __init__(self,prod,weight): # order object gets to have this attributes.
+			self.prod = prod
+			self.weight = weight
+			self.cost = self.weight * 10
+
+					
+			
+```
+
+Second, I have to determine how I’ll compute the order costs and totals in order to define the catalog rendering. I have a class that molds object with `prod, weight, cost` attributes. `Order` class has a `Total` method that accumulates each product’s total cost. Next, I have to determine how the user will be able to make a choice and relay the user selections. `# In this section I've opted for using a sql database because it will be simpler to integrate the data and have an aggregated platform.`
+
+Third, I’ll display the product catalog most probably in dictionary form, parse it then allow users to make selections without creating many lines of code using `if...else` conditional statements. User input variables will be used as reference objects in the dictionary iteration and conditional statements. 
+
+```python
```

Comment:
To make this program work, I have to break it down to small functions. Each function serving a particular process.